Makefile

```diff
@@ -145,6 +145,13 @@ scoop: all
 	@ echo "Running minimal example with SCOOP"
 	python -m scoop -n 2 rmg.py -v testing/scoop/input.py
 
+update: all
+	@ echo "Updating RMG-py"
+	git stash || true
```

Comment:
If this is made for people who forget, we also don’t know they didn’t forget to checkout main before updating. Don’t know what to do about it, just bringing this up